src/Shared/SFA.DAS.SharedOuterApi/Configuration/FeatureToggles.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SFA.DAS.SharedOuterApi.Configuration
+{
+    public class FeatureToggles
+    {
+        public bool RoatpStandardsEnabled { get; set; }
```

Comment:
Why is the feature toggle referring to "standards" when we are getting "Providers". rename this and modify the config please?

Reply:
Done